IMC_WebDev/FromDayToNight-CreatingInteractiveColorPalettesWithJavaScript/blog.md

```diff
@@ -0,0 +1,394 @@
+
+```css
+...
+button {
+	...
+    border-style: solid;
+    border-width: 2px;
+    border-color: #FF8080;
+    border-radius: 10px;
+}
+...
+```
+
+For all the border properties to show up, the `border-style` attribute first must be set. Above, we set our buttons' borders to `solid` that will result in a solid colored border. Feel free to experiment with different kinds of borders from this [CSS Border Styles list](https://www.w3schools.com/css/css_border.asp). As for the additional border attributes, `border-width` and`border-color` adjust the border's thickness and color, respectively, and `border-radius` adjusts the roundness of the corners of the border—this particular property allows up to four values, each value accessing each corner of the element.
+
+Finally, we will change the font of the text of our buttons with the last two following attributes:
+
+```css
+...
+button {
+	...
+    font-size: 15px;
+    font-family: "Consolas";
+}
+...
+```
+
+The `font-size` attribute will adjust the size of the buttons' font. Also, feel free to customize your buttons' font by testing out fonts from this [CSS font list](https://www.w3schools.com/cssref/css_websafe_fonts.asp) and setting it to your `font-family` attribute. You can also add additional attributes, such as `font-style` and `font-weight`, to stylize your buttons even more. 
+
+This is what our completed `button` element should look like:
+
+```css
+...
+button {
+    position: relative;
+    top: -40px;
+    right: -360px;
+    background-color: #FFFFFF;
+
+    border-style: solid;
+    border-width: 2px;
+    border-color: #FF8080;
+    border-radius: 10px;
+
+    font-size: 15px;
+    font-family: "Consolas";
+}
+...
+```
+
+This is what our Tiny City should like so far:
+
+![css_code_2](photos/css_code_2.jpg)
+
+
+
+## Adding Interactivity with JavaScript
+
+### JavaScript Code
+
+In order to be able to code in JavaScript in our HTML file, we will insert a set of `<script></script>` tags into the `body` element of our HTML code, underneath the code we added above:
+
+```html
+...
+<!--HTML-->
+  <body>
+      <button>day</button>
+      <button>night</button>
+      
+      <div id = "sun"></div>   
+      <body id = "body"></body> 
+    	
+      <script>
+	  </script>
+...
+```
+
+If we want to manipulate the colors of the sun/moon element and the background, we must retrieve our `sun` and `body` elements, respectively, by calling on their `id`. 
+
+To do this, we will declare a `var` or variable—`circle` for the `sun` element and `sky` for the `body` element— and then access each element by using the `document` method `getElementById()`:
+
+```js
+...
+<script>
+    var circle = document.getElementById("sun");
+    var sky = document.getElementById("body");
+</script>
+...
+```
+
+Since we already established the `moon` element for our planned color palette change, we should also choose the color we want to our night sky. 
```

Comment:
"... the color we want" our night sky to be? It sounds a little off the way you've said it